lms/djangoapps/branding/api.py

```diff
@@ -368,6 +369,19 @@ def get_base_url(is_secure):
     return _absolute_url(is_secure=is_secure, url_path="")
 
 
+def get_enterprise_customer_logo_url(**kwargs):
```

Comment:
We discussed the use of `**kwargs` on a different PR -- it's better to be explicit about what these functions are willing to accept for arguments.